test/postman/osmt_apitest.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+
+set -eu
+
+declare apitest_env_file
+
+declare OKTA_URL
+declare OKTA_USERNAME
+declare OKTA_PASSWORD
+
+function setup_environment() {
```

Comment:
For this test suite, we should expect that the user / env already has node and npm, and just verify that they are available. We should not take responsibility to install them. Installing newman locally makes OK sense to me, but we can't test that with a `which`.

Reply:
If we want a certain set of npm dependencies, then we should have a `package.json`, and call `npm install` on it. Please look at the arguments for saving dev dependencies.